Add blockly dev server start & Document npm run dev (#1888)
openhab/openhab-webui: bundles/org.openhab.ui/CONTRIBUTING.md

* `npm start` - run the development server (see below)
* `npm run build-prod` - build web app for production (note: no need to prepare a production version when submitting a PR, the build server will do it)
* `npm run build-cordova-prod` - build cordova's `www` folder from source and build a Cordova app

> * `npm run dev` - run server in dev mode for development using webpack.config.js

Reviewer: ```suggestion
* `npm run dev` - run the development server (same as above)
```

> ^* `npm run dev:blockly` - run the development server with Blockly source-maps (allows Blockly debugging)

Reviewer: ```suggestion
* `npm run dev:blockly` - run the development server with Blockly source-maps (allows Blockly debugging)
```

> ^* `npm run dev:blockly` - run server in dev mode for development using webpack.config.js and blockly source-maps

Reviewer: ```suggestion
* `npm run dev:blockly` - run the development server with Blockly source-maps (allows Blockly debugging)
```
